api-catalog-ui/frontend/src/components/Wizard/configs/wizard_categories.jsx

```diff
@@ -1,11 +1,11 @@
-/**
```

Comment:
Shouldn't we also keep this documentation after the License?

Reply:
Remove by eslint/header autofix, added comment back